src/pybind/mgr/orchestrator/module.py

```diff
@@ -1336,7 +1336,8 @@ def _daemon_add_osd(self,
         usage = """
 Usage:
   ceph orch daemon add osd host:device1,device2,...
-  ceph orch daemon add osd host:data_devices=device1,device2,db_devices=device3,osds_per_device=2,...
+  ceph orch daemon add osd host:data_devices=device1,device2,db_devices=device3,osds_per_device=2[,encrypted=true|True|1]
+  ceph orch daemon add osd host:data_devices=device1[,encrypted=false|False|0]
```

Comment:
```suggestion
  ceph orch daemon add osd host:data_devices=device1,device2,db_devices=device3,osds_per_device=2,[encrypted=true|True|1]
  ceph orch daemon add osd host:data_devices=device1,[encrypted=false|False|0]
```
unless the comma is within the brackets for a specific reason

Reply:
The comma was put inside the brackets since if we don't use the encrypted flag (which is an optional flag) there is no reason for the comma to be part of the command. I agree it looks better outside of the brackets though :)

Reply:
I'm not tied to it either way. I can merge it like this if you prefer it (after testing and such)

Reply:
Lets commit your suggestion, Its less confusing